driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -234,6 +241,23 @@ private void setFinalResult(
     }
   }
 
+  private void logServerWarnings(List<String> warnings) {
+    // log the warnings if they have NOT been disabled (double negative)
+    if (!Boolean.getBoolean(RequestHandler.DISABLE_QUERY_WARNING_LOGS)) {
```

Comment:
Since you already guarded the call to this method by a few conditional checks above, I'd suggest that you add this one to the `if` block above too.

Reply:
I think I had it that way at some point. Good call.